freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_backup

```diff
@@ -83,10 +92,19 @@ doLog(){
     echo "`date "+%Y-%m-%dT%H:%M:%SZ"` $type_of_msg ""$msg" >> $LOGFILE
 }
 
+doStatus(){
+    type_of_msg=$(echo $*|cut -d" " -f1)
+    msg=$(echo "$*"|cut -d" " -f2-)
+    [[ $type_of_msg == INFO ]] && type_of_msg="INFO " # one space for aligning
```

Comment:
so don't we need the same for error?

Reply:
No.  This code is just adding an addition space next to INFO so the logging lines up visually.